doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -1,9 +1,7 @@
 ﻿# Release notes
 
 ### Features
-- Support for `EmailManager.ShowComposeNewEmailAsync`
-- Add support for `StorageFolder.CreateFileAsync(string path)`
-- Add support for ApplicationViewTitleBar.BackgroundColor on WASM
+- Add support for Windows.ApplicationModel.DataTransfer.DataPackage.RequestedOperation.Copy
```

Comment:
You accidentally removed several release notes here

Reply:
Strange. I didn't remove anything, I just open relenasenotes and add one line, saving results to new branch. Seems like older version?
Corrected.